Hestia/Hestia/frontend/Auth0/ServerSelectScreen/ViewControllerServerList.cs

```diff
@@ -81,7 +81,7 @@ bool ShouldPerformSegue()
                 {
                     Console.WriteLine("Exception while getting devices from local server");
                     Console.WriteLine(ex);
-                    new WarningMessage("Could not fetch devices", "The server information is invalid. Did you forget to include https://?", this);
+                    WarningMessage.Display("Could not fetch devices", "The server information is invalid. Did you forget to include https://?", this);

```

Comment:
`Did you forget to include https://` should no longer be in this warning message.

Reply:
This is the warning message that is displayed when the servers on the Webserver do not have https:// (what I first tried, because I didn't know it should be included), or can the NetworkHandler now handle them without?

Reply:
Ok, it's fine then. I thought this was for a local server.